packages/backend-defaults/src/entrypoints/cache/CacheManager.ts

```diff
@@ -139,37 +139,39 @@ export class CacheManager {
   }
 
   private createRedisStoreFactory(): StoreFactory {
-    const KeyvRedis = require('@keyv/redis');
-    let store: typeof KeyvRedis | undefined;
+    const KeyvRedis = require('@keyv/redis').default;
+    const stores: Record<string, typeof KeyvRedis> = {};
```

Comment:
During initialization the Keyv instead sets the namespace on the provider adapter: https://github.com/jaredwray/keyv/blob/main/packages/keyv/src/index.ts#L181

This means that reusing a single adapter / connection is not possible for multiple plugins, because each new initialization would override the previous namespace.

This seems to be the case in older versions of the library as well: https://github.com/jaredwray/keyv/blob/2024-05-04/packages/keyv/src/index.js#L63. So I have no idea how it worked before regardless of the useRedisSets option :thinking: